### ISSUE TITLE
Windows 10 issue when trying to show the management studio window - it's invisible

### ISSUE THREAD

**Issue by JuliusSweetland** [JuliusSweetland/OptiKey#85]
"   When I try to open the management console ( by press Alt-M ) on
   Windows 10 64bit the management console doesn't actually show the
   window properly, I can see it if I hover over the icon, but
   clicking on it doesn't actually show the window."

![screenshot](https://cloud.githubusercontent.com/assets/4722561/9810029/2ed08c48-5865-11e5-9916-6abdb2a16702.jpg)

Video of problem: http://www.promixis.com/downloads/temp/capture-1.mp4

**Comment by danielmoncada** [comment]
Interesting.. this works fine for me.

Windows 10 Professional N 64bit

**Comment by JuliusSweetland** [comment]
I haven't upgraded to Win10 so I can't test at the moment, but my dad saw this issue once. He told me that it didn't recur, so it's definitely there, but intermittent.

**Comment by JuliusSweetland** [comment]
I have some sample user.settings files from a user who experienced this problem - not sure which one is relevent to this problem:

Comments below...

**Comment by JuliusSweetland** [comment]
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <userSettings>
        <JuliusSweetland.OptiKey.Properties.Settings>
            <setting name="KeySelectionTriggerSource" serializeAs="String">
                <value>Fixations</value>
            </setting>
            <setting name="SettingsUpgradeRequired" serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="PointsMousePositionSampleInterval" serializeAs="String">
                <value>00:00:00.0130000</value>
            </setting>
            <setting name="PointsSource" serializeAs="String">
                <value>MousePosition</value>
            </setting>
            <setting name="PointTtl" serializeAs="String">
                <value>00:00:00.1500000</value>
            </setting>
            <setting name="PointSelectionTriggerFixationCompleteTime" serializeAs="String">
                <value>00:00:01.2500000</value>
            </setting>
            <setting name="Language" serializeAs="String">
                <value>BritishEnglish</value>
            </setting>
            <setting name="KeySelectionTriggerFixationCompleteTime" serializeAs="String">
                <value>00:00:01.2500000</value>
            </setting>
            <setting name="MainWindowFloatingSizeAndPosition" serializeAs="String">
                <value>280,175,1120,700</value>
            </setting>
            <setting name="ToastNotificationSecondsPerCharacter" serializeAs="String">
                <value>0.05</value>
            </setting>
            <setting name="PointSelectionTriggerSource" serializeAs="String">
                <value>Fixations</value>
            </setting>
            <setting name="KeySelectionTriggerIncompleteFixationTtl" serializeAs="String">
                <value>00:00:00.7500000</value>
            </setting>
            <setting name="KeySelectionTriggerFixationLockOnTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="PointSelectionTriggerFixationLockOnTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="ProgressIndicatorBehaviour" serializeAs="String">
                <value>FillPie</value>
            </setting>
            <setting name="MainWindowFullDockThicknessAsPercentageOfScreen"
                serializeAs="String">
                <value>50</value>
            </setting>
            <setting name="MainWindowDockPosition" serializeAs="String">
                <value>Top</value>
            </setting>
            <setting name="MainWindowDockSize" serializeAs="String">
                <value>Full</value>
            </setting>
            <setting name="MainWindowState" serializeAs="String">
                <value>Docked</value>
            </setting>
            <setting name="StartupKeyboard" serializeAs="String">
                <value>Alpha</value>
            </setting>
        </JuliusSweetland.OptiKey.Properties.Settings>
    </userSettings>
</configuration>

**Comment by JuliusSweetland** [comment]
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <userSettings>
        <JuliusSweetland.OptiKey.Properties.Settings>
            <setting name="KeySelectionTriggerSource" serializeAs="String">
                <value>Fixations</value>
            </setting>
            <setting name="SettingsUpgradeRequired" serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="PointsMousePositionSampleInterval" serializeAs="String">
                <value>00:00:00.0130000</value>
            </setting>
            <setting name="PointsSource" serializeAs="String">
                <value>MousePosition</value>
            </setting>
            <setting name="PointTtl" serializeAs="String">
                <value>00:00:00.1500000</value>
            </setting>
            <setting name="PointSelectionTriggerFixationCompleteTime" serializeAs="String">
                <value>00:00:01.2500000</value>
            </setting>
            <setting name="Language" serializeAs="String">
                <value>BritishEnglish</value>
            </setting>
            <setting name="KeySelectionTriggerFixationCompleteTime" serializeAs="String">
                <value>00:00:01.2500000</value>
            </setting>
            <setting name="MainWindowFloatingSizeAndPosition" serializeAs="String">
                <value>280,175,1120,700</value>
            </setting>
            <setting name="ToastNotificationSecondsPerCharacter" serializeAs="String">
                <value>0.05</value>
            </setting>
            <setting name="PointSelectionTriggerSource" serializeAs="String">
                <value>Fixations</value>
            </setting>
            <setting name="KeySelectionTriggerIncompleteFixationTtl" serializeAs="String">
                <value>00:00:00.7500000</value>
            </setting>
            <setting name="KeySelectionTriggerFixationLockOnTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="PointSelectionTriggerFixationLockOnTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="ProgressIndicatorBehaviour" serializeAs="String">
                <value>FillPie</value>
            </setting>
            <setting name="MainWindowFullDockThicknessAsPercentageOfScreen"
                serializeAs="String">
                <value>50</value>
            </setting>
            <setting name="MainWindowDockPosition" serializeAs="String">
                <value>Top</value>
            </setting>
            <setting name="MainWindowDockSize" serializeAs="String">
                <value>Full</value>
            </setting>
            <setting name="MainWindowState" serializeAs="String">
                <value>Docked</value>
            </setting>
            <setting name="StartupKeyboard" serializeAs="String">
                <value>Alpha</value>
            </setting>
        </JuliusSweetland.OptiKey.Properties.Settings>
    </userSettings>
</configuration>

**Comment by JuliusSweetland** [comment]
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <userSettings>
        <JuliusSweetland.OptiKey.Properties.Settings>
            <setting name="KeySelectionTriggerKeyboardKeyDownUpKey" serializeAs="String">
                <value>Insert</value>
            </setting>
            <setting name="MultiKeySelectionTriggerStopSignal" serializeAs="String">
                <value>NextHigh</value>
            </setting>
            <setting name="KeySelectionTriggerMouseDownUpButton" serializeAs="String">
                <value>Left</value>
            </setting>
            <setting name="KeySelectionTriggerSource" serializeAs="String">
                <value>Fixations</value>
            </setting>
            <setting name="SettingsUpgradeRequired" serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="PointsMousePositionSampleInterval" serializeAs="String">
                <value>00:00:00.0130000</value>
            </setting>
            <setting name="PointsSource" serializeAs="String">
                <value>TheEyeTribe</value>
            </setting>
            <setting name="PointTtl" serializeAs="String">
                <value>00:00:00.1500000</value>
            </setting>
            <setting name="PointSelectionTriggerLockOnRadiusInPixels" serializeAs="String">
                <value>40</value>
            </setting>
            <setting name="PointSelectionTriggerFixationRadiusInPixels" serializeAs="String">
                <value>60</value>
            </setting>
            <setting name="PointSelectionTriggerFixationCompleteTime" serializeAs="String">
                <value>00:00:01.2500000</value>
            </setting>
            <setting name="MultiKeySelectionFixationMinDwellTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="MaxDictionaryMatchesOrSuggestions" serializeAs="String">
                <value>20</value>
            </setting>
            <setting name="MultiKeySelectionMaxDuration" serializeAs="String">
                <value>00:01:00</value>
            </setting>
            <setting name="Language" serializeAs="String">
                <value>BritishEnglish</value>
            </setting>
            <setting name="KeySelectionTriggerFixationCompleteTime" serializeAs="String">
                <value>00:00:01.2500000</value>
            </setting>
            <setting name="MainWindowFloatingSizeAndPosition" serializeAs="String">
                <value>280,175,1120,700</value>
            </setting>
            <setting name="ToastNotificationVerticalFillPercentage" serializeAs="String">
                <value>75</value>
            </setting>
            <setting name="ToastNotificationSecondsPerCharacter" serializeAs="String">
                <value>0.05</value>
            </setting>
            <setting name="Theme" serializeAs="String">
                <value>/Resources/Themes/Android_Dark.xaml</value>
            </setting>
            <setting name="Debug" serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="FontFamily" serializeAs="String">
                <value>/Resources/Fonts/#Roboto</value>
            </setting>
            <setting name="FontStretch" serializeAs="String">
                <value>Condensed</value>
            </setting>
            <setting name="FontWeight" serializeAs="String">
                <value>Light</value>
            </setting>
            <setting name="ScratchpadNumberOfLines" serializeAs="String">
                <value>2</value>
            </setting>
            <setting name="AutoAddSpace" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="AutoCapitalise" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="SpeechRate" serializeAs="String">
                <value>0</value>
            </setting>
            <setting name="SpeechVoice" serializeAs="String">
                <value />
            </setting>
            <setting name="SpeechVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="ErrorSoundFile" serializeAs="String">
                <value>Resources\Sounds\Tone2.wav</value>
            </setting>
            <setting name="InfoSoundFile" serializeAs="String">
                <value>Resources\Sounds\Tone1.wav</value>
            </setting>
            <setting name="KeySelectionSoundFile" serializeAs="String">
                <value>Resources\Sounds\Click1.wav</value>
            </setting>
            <setting name="MultiKeySelectionCaptureEndSoundFile" serializeAs="String">
                <value>Resources\Sounds\Falling1.wav</value>
            </setting>
            <setting name="MultiKeySelectionCaptureStartSoundFile" serializeAs="String">
                <value>Resources\Sounds\Rising1.wav</value>
            </setting>
            <setting name="PointSelectionTriggerSource" serializeAs="String">
                <value>Fixations</value>
            </setting>
            <setting name="KeySelectionTriggerIncompleteFixationTtl" serializeAs="String">
                <value>00:00:00.7500000</value>
            </setting>
            <setting name="KeySelectionTriggerFixationLockOnTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="PointSelectionTriggerFixationLockOnTime" serializeAs="String">
                <value>00:00:00.2500000</value>
            </setting>
            <setting name="ConversationOnlyMode" serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="CheckForUpdates" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="ShowSplashScreen" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="TETCalibrationCheckTimeSpan" serializeAs="String">
                <value>00:00:03</value>
            </setting>
            <setting name="MouseClickSoundFile" serializeAs="String">
                <value>Resources\Sounds\MouseClick1.wav</value>
            </setting>
            <setting name="MouseDoubleClickSoundFile" serializeAs="String">
                <value>Resources\Sounds\MouseDoubleClick1.wav</value>
            </setting>
            <setting name="MouseScrollSoundFile" serializeAs="String">
                <value>Resources\Sounds\MouseScroll1.wav</value>
            </setting>
            <setting name="CursorWidthInPixels" serializeAs="String">
                <value>60</value>
            </setting>
            <setting name="CursorHeightInPixels" serializeAs="String">
                <value>90</value>
            </setting>
            <setting name="MagnifySourcePercentageOfScreen" serializeAs="String">
                <value>10</value>
            </setting>
            <setting name="MagnifyDestinationPercentageOfScreen" serializeAs="String">
                <value>60</value>
            </setting>
            <setting name="ToastNotificationHorizontalFillPercentage" serializeAs="String">
                <value>75</value>
            </setting>
            <setting name="AutoCompleteWords" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="PointSelectionTriggerKeyboardKeyDownUpKey" serializeAs="String">
                <value>Insert</value>
            </setting>
            <setting name="PointSelectionTriggerMouseDownUpButton" serializeAs="String">
                <value>Left</value>
            </setting>
            <setting name="SuppressModifierKeysWhenInMouseKeyboard" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="ErrorSoundVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="InfoSoundVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="KeySelectionSoundVolume" serializeAs="String">
                <value>50</value>
            </setting>
            <setting name="MouseClickSoundVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="MouseDoubleClickSoundVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="MouseScrollSoundVolume" serializeAs="String">
                <value>75</value>
            </setting>
            <setting name="MultiKeySelectionCaptureEndSoundVolume" serializeAs="String">
                <value>50</value>
            </setting>
            <setting name="MultiKeySelectionCaptureStartSoundVolume" serializeAs="String">
                <value>50</value>
            </setting>
            <setting name="ProgressIndicatorBehaviour" serializeAs="String">
                <value>FillPie</value>
            </setting>
            <setting name="ProgressIndicatorResizeStartProportion" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="ProgressIndicatorResizeEndProportion" serializeAs="String">
                <value>20</value>
            </setting>
            <setting name="KeySelectionTriggerFixationResumeRequiresLockOn"
                serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="MainWindowFullDockThicknessAsPercentageOfScreen"
                serializeAs="String">
                <value>50</value>
            </setting>
            <setting name="MainWindowDockPosition" serializeAs="String">
                <value>Top</value>
            </setting>
            <setting name="MainWindowDockSize" serializeAs="String">
                <value>Full</value>
            </setting>
            <setting name="MainWindowState" serializeAs="String">
                <value>Docked</value>
            </setting>
            <setting name="MainWindowCollapsedDockThicknessAsPercentageOfFullDockThickness"
                serializeAs="String">
                <value>20</value>
            </setting>
            <setting name="MultiKeySelectionEnabled" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="MouseDownSoundFile" serializeAs="String">
                <value>Resources\Sounds\MouseDown1.wav</value>
            </setting>
            <setting name="MouseUpSoundFile" serializeAs="String">
                <value>Resources\Sounds\MouseUp1.wav</value>
            </setting>
            <setting name="MouseDownSoundVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="MouseUpSoundVolume" serializeAs="String">
                <value>100</value>
            </setting>
            <setting name="StartupKeyboard" serializeAs="String">
                <value>Alpha</value>
            </setting>
            <setting name="SuppressAutoCapitaliseIntelligently" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="MainWindowMinimisedPosition" serializeAs="String">
                <value>SameAsDockedPosition</value>
            </setting>
        </JuliusSweetland.OptiKey.Properties.Settings>
    </userSettings>
</configuration>
```

**Comment by JuliusSweetland** [comment]
Testing on Windows 10 machine. Problem occurs. Installed Snoop to investigate, which installed .Net framework 3.0 and 3.5 (I think) = problem gone.

**Comment by Gadal** [comment]
Looks like the UIAccess flag gets ignored in some circumstances to prevent clickjacking.

Some things that you might try:
- Run OptiKey as an administrator.
- Move the executable to a trusted directory like /Program Files
- Digitally sign the executable (makecert + signtool)

**Comment by JuliusSweetland** [comment]
Thanks Dustin. I've tried all of those things. UIAccess works in Windows
8.1, it's just somehow connected to the child manager window not being
rendered in Windows 10.

**Comment by Gadal** [comment]
It's probably the NSA's fault.

**Comment by JuliusSweetland** [comment]
Probably, ha

**Comment by JuliusSweetland** [comment]
Fixed in v2.0.10
